exporter/views.py

```diff
@@ -92,9 +100,10 @@ class ExportCampaignToBagit(TemplateView):
     @method_decorator(login_required)
     def get(self, request, *args, **kwargs):
         campaign_slug = self.kwargs["campaign_slug"]
-        assets = get_latest_transcription_data(campaign_slug)
+        assets = get_latest_transcription_data(campaign_slug, True)
 
         export_base_dir = os.path.join(settings.SITE_ROOT_DIR, "tmp", campaign_slug)
```

Comment:
The old code here was unsafe and we'd get some level of corruption if, say, someone double-clicked the link. I would use [`mkdtemp`](https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp) or the context-manager form of [`TemporaryDirectory`](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) with using a prefix, which almost certainly is not `SITE_ROOT_DIR`.